backend/src/main/java/heartbeat/service/pipeline/buildkite/BuildKiteService.java

```diff
@@ -63,6 +77,17 @@ public BuildKiteResponse fetchPipelineInfo() {
 			log.error("[BuildKite] Failed when call BuildKite", e);
 			throw new RequestFailedException(e);
 		}
+		catch (NoPermissionException e) {
```

Comment:
Maybe we can use `if` directly, in that case, we can reduce one throw/catch.
Or we can add other handler to handle `NoPermissionException`.